code/game/gamemodes/clock_cult/clock_items/clockwork_slab.dm

```diff
@@ -327,57 +329,238 @@
+			surgically removed before they are an eligible convert. <i>Note:</i> The captain is <i>never</i> an eligible convert and should instead be killed or imprisoned. If security \
+			begins administering mindshield implants, this will greatly inhibit conversion. Also note that mindshield implants can be broken by a sigil of accession automatically, but \
+			the sigil will disappear.<br><br>"
+			dat += "<font color=#BE8700><b>Converting Silicons:</b></font> Due to their robotic nature, silicons are generally more predictable than humans in terms of conversion. \
+			However, they are also much, much harder to subdue, especially cyborgs. The easiest way to convert a cyborg is by using a flash or a sigil of transgression to stun it, \
+			then very quickly using Geis to restrain them. If you stack a sigil of transgression and one of the conversion sigils, a crossing cyborg will be stunned and helpless to \
+			escape in time before the other sigil converts them.<br><br>"
+			dat += "Converting AIs is very often the hardest task of the cult, and has been the downfall of countless successful servants. Their omnipresence across the station, \
+			coupled with their secure location and ability to lock themselves securely, makes them a powerful target. However, once the AI itself is reached, it is usually completely \
+			helpless to resist its own conversion. A very common tactic is to take advantage of a converted cyborg to rush the AI before it is able to react.<br><br>"
+			dat += "Even once an AI is converted, care must be taken to ensure that it remains hidden. Not only does the AI's core become brassy and thus obvious to an outside \
+			observer, but <i>the AI loses the ability to speak in anything but Ratvarian.</i> For this reason, it has to remain completely silent over common radio channels if stealth \
+			is at all a priority. This is suspicious and will rapidly lead to the crew checking on it, which usually results in the cult's outing. It is, however, necessary to convert \
+			all AIs present on the station before the Ark becomes invokable, so this must be done at some point.<br><br>"
+			dat += "<font color=#BE8700><b>Converting the Station:</b></font> Converted objects all serve a purpose and are important to the cult's success. To convert objects, \
+			a servant needs to use a <b>replica fabricator,</b> a handheld tool that uses power to convert objects to clockwork versions. Different clockwork objects have different \
+			effects and are often crucial. The most noteworthy are <b>clockwork walls,</b> which automatically \"link\" to any nearby tinkerer's caches, causing them to <b>slowly \
+			generate components.</b> This is incredibly useful for obvious reasons, and creating a clockwork wall near every tinkerer's cache should be prioritized. Clockwork floors \
```

Comment:
it is my curse to prefer capitalization for anything related to the cult or that is a cult object